src/gateways/createBookConfigLoader.ts

```diff
@@ -0,0 +1,48 @@
+import { BOOKS } from '../config';
+import { acceptStatus } from '../helpers/fetch';
+import Sentry, { Severity } from '../helpers/Sentry';
+
+export interface BookVersion {
+  defaultVersion: string;
+}
+
+interface BookConfig {
+  [key: string]: BookVersion;
+}
+
+interface ReleaseJsonStructure {
+  books: BookConfig;
+  code: string;
+  id: string;
+}
+
+const cachedBooks = { ...BOOKS } as BookConfig;
```

Comment:
this is the type that the books object already has, why do you need to redeclare `BookConfig` and `BookVersion` here?

Reply:
yea, actually it is not needed to point it here `const cachedBooks = { ...BOOKS } as BookConfig;`, but these interfaces are showing what data types the function returns.

Reply:
in the case of the function you can just use `typeof BOOKS` right?